lib/msal-common/src/error/ClientAuthError.ts

```diff
@@ -193,6 +193,10 @@ export const ClientAuthErrorMessage = {
     keyIdMissing: {
         code: "key_id_missing",
         desc: "A keyId value is missing from the requested bound token's cache record and is required to match the token to it's stored binding key."
+    },
+    invalidRequest: {
+        code: "invalid_request",
+        desc: "The request body must contain the following parameter: 'username'"
```

Comment:
This is not correct. The description comes from ESTS.

Reply:
Thanks for your helpful feedback, removed the custom descriptions @bgavrilMS